disperser/Makefile

```diff
@@ -63,4 +66,32 @@ run_encoder: build_encoder
   --disperser-encoder.log.level-file debug
 
 run_dataapi_tests:
-	go test -v disperser/dataapi
\ No newline at end of file
+	go test -v disperser/dataapi
+
+run_blobapi: build_blobapi
```

Comment:
I would recommend removing most of these args related to buckets, contracts and anything else that would be env specific and force the user to export those values via `.env` or other script that can be sourced.